assets/dev/js/editor/document/hooks/ui/delete/index.js

```diff
@@ -2,3 +2,4 @@
 
 export { ColumnIsEmpty } from './column-is-empty';
 export { DeleteSectionIsFull } from './section-is-full';
```

Comment:
Those two old hooks does not following the convention.
they need to be located at this path:
`document/elements/delete`.